wear/src/main/java/io/homeassistant/companion/android/home/MainViewModel.kt

```diff
@@ -278,6 +279,8 @@ class MainViewModel @Inject constructor(
         isEnabled: Boolean
     ) {
         sensorDao.setSensorsEnabled(listOf(basicSensor.id), isEnabled)
+        sensorDao.updateLastSentStateAndIcon(basicSensor.id, null, null)
```

Comment:
This line isn't necessary because the sensor DAO will already [set the state and icon to null](https://github.com/home-assistant/android/blob/93844ffea03be01e35c618114073b0e26fd47f39/common/src/main/java/io/homeassistant/companion/android/database/sensor/SensorDao.kt#L88).

Reply:
I think this got added when I was discovered the HA core issue, will remove it